test/browser_tests/shared_objects/secondary-navigation.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var _secondaryNav = element( by.css( '.o-secondary-navigation' ) );
+
+var secondaryNav = {
+
+  secondaryNav: _secondaryNav,
+
+  expandableTarget:
+  _secondaryNav.element( by.css( '.m-expandable_target' ) ),
```

Comment:
I think we've been indenting values on a new line to make it clear they've been moved only because of the 80char limit